chat/message/src/MessageLinks/MessageLinks.spec.tsx

```diff
@@ -0,0 +1,334 @@
+        expect(screen.getByText(link.children)).toBeInTheDocument();
+      });
+    });
+  });
+
+  describe('expand/collapse behavior', () => {
+    test('starts in collapsed state by default', () => {
+      const { container } = renderMessageLinks();
+      const toggleButton = screen.getByRole('button');
+      const contentRegion = container.querySelector('[role="region"]');
+
+      expect(toggleButton).toHaveAttribute('aria-expanded', 'false');
+      expect(toggleButton).toHaveAttribute(
+        'aria-label',
+        'Expand Related Resources',
+      );
+      expect(contentRegion).toHaveStyle({
+        height: '0px',
+        opacity: '0',
+        visibility: 'hidden',
+      });
+    });
+
+    test('expands when toggle button is clicked', async () => {
+      const { container } = renderMessageLinks();
+
+      const toggleButton = screen.getByRole('button');
+      const contentRegion = container.querySelector('[role="region"]');
+
+      await userEvent.click(toggleButton);
+
+      expect(toggleButton).toHaveAttribute('aria-expanded', 'true');
+      expect(toggleButton).toHaveAttribute(
+        'aria-label',
+        'Collapse Related Resources',
+      );
+
+      // Content should become visible (opacity and visibility change immediately, height animates)
+      await waitFor(() => {
+        expect(contentRegion).toHaveStyle({
+          opacity: '1',
+          visibility: 'visible',
+        });
+      });
+    });
+
+    test('collapses when toggle button is clicked again', async () => {
+      const { container } = renderMessageLinks();
+
+      const toggleButton = screen.getByRole('button');
+      const contentRegion = container.querySelector('[role="region"]');
+
+      // First click to expand
+      await userEvent.click(toggleButton);
+      await waitFor(() => {
+        expect(toggleButton).toHaveAttribute('aria-expanded', 'true');
+      });
+
+      // Second click to collapse
+      await userEvent.click(toggleButton);
+
+      expect(toggleButton).toHaveAttribute('aria-expanded', 'false');
+      expect(toggleButton).toHaveAttribute(
+        'aria-label',
+        'Expand Related Resources',
+      );
+      expect(contentRegion).toHaveStyle({
+        height: '0px',
+        opacity: '0',
+        visibility: 'hidden',
+      });
+    });
+
+    test('icon rotates when expanded/collapsed', async () => {
+      renderMessageLinks();
+
+      const toggleButton = screen.getByRole('button');
+
+      // Initially should be collapsed (aria-expanded="false")
+      expect(toggleButton).toHaveAttribute('aria-expanded', 'false');
+
+      // Click to expand
+      await userEvent.click(toggleButton);
+
+      // Should now be expanded with different styling
+      await waitFor(() => {
+        expect(toggleButton).toHaveAttribute('aria-expanded', 'true');
+        // The icon should have different classes applied for rotation
+        expect(toggleButton.className).toMatch(/leafygreen-ui/);
```

Comment:
Why do we need this test? I also don't think this test is correct since the className always starts with `leafygreen-ui`